content/blog/2021-07-tonic-0-5.md

```diff
@@ -0,0 +1,157 @@
+---
+date: "2021-07-09"
+title: "Announcing Tonic 0.5"
+description: "July 9, 2021"
+---
+
+We are pleased to announce version 0.5 of [Tonic], a native gRPC implementation in
+Rust. 0.5 is a big release and has been in the works for some time.
+
+Some key new features are:
+
+# grpc-web
+
+[grpc-web] allows web clients to connect to gRPC services over HTTP/1.1. The new
+crate [tonic-web] allows regular Tonic services to accept grpc-web requests
+without the need of an external proxy.
```

Comment:
The mention of both grpc-web and tonic-web here is confusing. What is what? What is new?

Reply:
grpc-web is the protocol, tonic-web is the crate that enables tonic to speak grpc-web. I see it might be confusing. I've tweaked the wording a bit. Hopefully its more clear.